e2e/app/aou-elements/xpath-defaults.ts

```diff
@@ -2,11 +2,11 @@ import TextOptions from './text-options';
 
 function textXpathHelper(opts: TextOptions) {
   if (opts.text) {
-    return `text()="${opts.text}" or @aria-label="${opts.text}" or @placeholder="${opts.text}"`;
+    return `text()='${opts.text}' or @aria-label='${opts.text}' or @placeholder='${opts.text}'`;
```

Comment:
nit: can you declare a string for `'${opts.text}'` and use it in all three places?